include/exadg/incompressible_navier_stokes/driver.cpp

```diff
@@ -257,7 +257,7 @@ Driver<dim, Number>::solve() const
 
     if(application->get_parameters().ale_formulation == true)
     {
-      while(not time_integrator->finished())
+      while(not(time_integrator->finished()))
```

Comment:
this was not done at some place, I will go back and search for it.

Reply:
ok; I marked some places now (comments above). The general rule is that if whatever follows `not` is super simple, we omit the brackets (like `not true` ), but if its more complex, we put them (like `not( cond_evaluator->evaluate_cond())` or `not(true and (false or true))`), right?

Reply:
agree